Source/SwiftyDropbox/Shared/Handwritten/DropboxClientsManager.swift

```diff
@@ -56,15 +56,18 @@ public class DropboxClientsManager {
         )
         DropboxOAuthManager.sharedOAuthManager = oAuthManager
 
-        setUpAuthorizedClient(
-            transportClient: transportClient,
-            backgroundTransportClient: backgroundTransportClient,
-            sessionConfiguration: nil,
-            backgroundSessionConfiguration: nil,
-            oAuthManager: oAuthManager,
-            oauthSetupIntent: oauthSetupIntent,
-            requestsToReconnect: requestsToReconnect
-        )
+        if let token = token(for: oauthSetupIntent.userKind, using: oAuthManager) {
+            setUpAuthorizedClient(
```

Comment:
Should there be an error logged for no token?

Reply:
It's actually a valid state, the model here is that users should call the setup* function on every launch. Auth can be performed afterwards if needed.